### PR TITLE
Provides the first steps for the new Tycho UI

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/web/templates/DefaultGlobalContextExtender.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ public void collectTemplate(Collector parameterCollector) {
         parameterCollector.collect("lang", NLS.getCurrentLang());
         parameterCollector.collect("contentHelper", ContentHelper.INSTANCE);
         parameterCollector.collect("wondergemRoot", wondergemRoot, String.class);
+        parameterCollector.collect("tychoRoot", wondergemRoot, String.class);
```
Comment (sabieber) [inline src/main/java/sirius/web/templates/DefaultGlobalContextExtender.java:64]: Should this be the same value as wondergem root?

**Review by sabieber — CHANGES_REQUESTED** [review]

File: src/main/resources/default/assets/tycho/lib.css.pasta [hunk 2]
```diff
@@ -0,0 +1 @@
+___include("/assets/tycho/libs/bootstrap/css/bootstrap.css")
```
Comment (jmuscireum) [inline src/main/resources/default/assets/tycho/lib.css.pasta:1]: Should be instead use the min versions for css and js?

**Review by bwiedmann — APPROVED** [review]
👍 🛫

File: src/main/java/sirius/web/templates/DefaultGlobalContextExtender.java (hunk 1 above)
Comment (andyHa) [reply src/main/java/sirius/web/templates/DefaultGlobalContextExtender.java:64]: yop for backwards compatibility

**Review by sabieber — APPROVED** [review]

**Review by idlira — APPROVED** [review]

**Review by bwiedmann — APPROVED** [review]

**Pull request merged**